analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/PublicMethodArgumentsShouldBeCheckedForNullTest.cs

```diff
@@ -53,6 +56,17 @@ public class PublicMethodArgumentsShouldBeCheckedForNullTest
                 .AddPaths("PublicMethodArgumentsShouldBeCheckedForNull.cs")
                 .Verify();
 
+        [TestMethod]
+        public void PublicMethodArgumentsShouldBeCheckedForNull_Assignment_State_Roslyn_CS()
+        {
+            var snippet = """
+                    arg = Unknown<string>();
```

Comment:
From the existing occurrences of `"""` in the `sonar-dotnet` codebase, I almost always see code indented one level w.r.t. the instruction, instead of two:
```cs
var snippet = """
    arg = Unknown<string>();
    Tag("AfterAssignment", arg);
    """;
```